packages/razzle-plugin-typescript/helpers.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
```

Comment:
Let's move this whole file to `razzle-dev-utils/pluginHelpers` because we'll probably reuse this all over the place